components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java

```diff
@@ -448,6 +522,21 @@ private void addModelDefinition(Node current, ModelImpl model, String type, bool
         }
     }
 
+    private boolean isChildNode(Node current, Node prevNode) {
+        boolean isChild = false;
+        if (prevNode.hasChildNodes()) {
+            NodeList childNodes = prevNode.getChildNodes();
+            for (int i = 0; i < childNodes.getLength(); i++) {
+                if (childNodes.item(i).getAttributes() != null && current.getAttributes()
+                        .getNamedItem(SOAPToRESTConstants.NAME_ATTRIBUTE).getNodeValue().equals(childNodes.item(i)
+                                .getAttributes().getNamedItem(SOAPToRESTConstants.NAME_ATTRIBUTE).getNodeValue())) {
+                    isChild = true;
```

Comment:
can we just return true? If it is true, no need to loop remaining objects right?

Reply:
Yes..Fixed.